test/CMakeLists.txt

```diff
@@ -139,6 +139,10 @@ if(LINUX AND TARGET sdl-build-options)
     target_include_directories(testevdev BEFORE PRIVATE ${SDL3_SOURCE_DIR}/src)
 endif()
 
+if(QNX)
+  target_link_libraries(testautomation PRIVATE m)
```

Comment:
Are the missing math symbols coming from the `testautomation*.c` sources or from withing `SDL3_test`/`SDL3-static`?
If it's the latter, then the main cmake script should be modified to add `m` to the interface link libraries.

Reply:
It's needed by `testautomation_math.c` (`is_inf()` etc.)